src/v0/destinations/braze/deleteUsers.js

```diff
@@ -22,6 +22,7 @@ const userDeletionHandler = async (userAttributes, config) => {
   // Endpoints different for different data centers.
   // DOC: https://www.braze.com/docs/user_guide/administrative/access_braze/braze_instances/
   let endPoint;
+  const endpointPath = '/users/delete'; // TODO: to handle for destinations dynamically by extracting from endpoint
   const dataCenterArr = dataCenter.trim().split('-');
   if (dataCenterArr[0].toLowerCase() === 'eu') {
     endPoint = 'https://rest.fra-01.braze.eu//users/delete';
```

Comment:
can we remove extra / in endPoint = 'https://rest.fra-01.braze.eu//users/delete'